sfja/src/index.js

```diff
@@ -1,28 +1,31 @@
 import React from 'react';
-import './index.css';
-import Admin from './Admin';
-import Form from './Form';
-import * as serviceWorker from './serviceWorker';
-import {Route, BrowserRouter as Router, Redirect} from 'react-router-dom';
 import ReactDOM from 'react-dom';
-import Header from './components/Header';
+import App from './App';
+import * as serviceWorker from './serviceWorker';
+import {Auth0Provider} from './react-auth0-spa';
+import config from './auth_config.json';
+import history from './utils/history';
 
-const routing = (
```

Comment:
woah a lot changed here
can we make sure /admin and /form/:key and /header are preserved?